test/comparePayloads.spec.js

```diff
@@ -0,0 +1,284 @@
+// Require Node.js Dependencies
+import { it } from "node:test";
+import assert from "node:assert";
+import { join, dirname } from "node:path";
+import { fileURLToPath } from "node:url";
+import { readFileSync } from "node:fs";
+
+// Require Internal Dependencies
+import { comparePayloads } from "../index.js";
+
+// CONSTANTS
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const FIXTURE_PATH = join(__dirname, "fixtures", "scannerPayloads");
+const payload = JSON.parse(readFileSync(join(FIXTURE_PATH, "/payload.json"), "utf8"));
```

Comment:
```suggestion
const kPayload = JSON.parse(readFileSync(join(FIXTURE_PATH, "/payload.json"), "utf8"));
```

We can even be more explicit (maybe overkill)
```suggestion
const kPayloadToBeCompared = JSON.parse(readFileSync(join(FIXTURE_PATH, "/payload.json"), "utf8"));
```